src/interfaces/plugins.ts

```diff
@@ -17,6 +17,8 @@ import Debug from 'debug'
 import { Request, Response } from 'express'
 const debug = Debug('signalk:interfaces:plugins')
 // @ts-ignore
+import SerialPort from 'serialport'
+// @ts-ignore
```

Comment:
I think there are types for serialport, installing them should get rid of the need for the ignore and then we can return something other than any below